.github/workflows/test-ci-windows.yml

```diff
@@ -0,0 +1,333 @@
+name: CI/CD for IIS Module
+
+on:
+  push:
+    branches:
+      - v2/test-ci-windows
```

Comment:
I'm not sure this branch is a good choice. You should use `v2/master` or keep this empty as we use this on Linux [CI](https://github.com/owasp-modsecurity/ModSecurity/blob/v3/master/.github/workflows/ci.yml#L3-L5).